src/components/HCPSignupFinish.js

```diff
@@ -36,22 +36,37 @@ class HCPSignupFinish extends React.Component {
     this.props.backend
       .continueSignup(url, this.state.confirmEmail ? this.state.email : null)
       .then(() => {
-        this.props.backend.getRequests(this.state.dropsite).then((requests) => {
-          if (requests?.length) {
-            this.props.history.push(
-              routeWithParams(Routes.DROPSITE_NEW_ADMIN, {
-                dropsite: this.state.dropsite,
-              }),
-            );
-            return;
-          }
+        this.props.backend
+          .getDropSites(this.state.dropsite)
+          .then((data) => {
+            if (!data.dropSiteAddress) {
```

Comment:
why data.dropSiteAddress vs data?

Reply:
We'll want to make sure DROPSITE_NEW_ADMIN merges / doesn't clobber if it's only address that's missing.

Reply:
my thinking was that if they had already entered the location details they wouldn't have to see this screen again but could choose to update this information via the facility page. the earlier process of creating a placeholder dropsite will always render data as truthy here. thoughts?

to the second note; we merge the data for this form with what is currently in firebase in the facility page PR 😄

Reply:
ah, right, makes sense! thanks!